recipes/ignition-math/all/conandata.yml

```diff
@@ -1,10 +1,26 @@
 sources:
   "6.7.0":
-    url: "https://github.com/ignitionrobotics/ign-math/archive/ignition-math6_6.7.0.zip"
-    sha256: "5427a17de6ef9f60ee497b9a170a4e92517de9496781dac91f50cdbeb3364b22"
+    url: "https://github.com/ignitionrobotics/ign-math/archive/ignition-math6_6.7.0.tar.gz"
+    sha256: "8456af51cbb128d7468d65b55124af7a235f052214ac2a239c3f23197416f2d2"
+  "6.8.0":
+    url: "https://github.com/ignitionrobotics/ign-math/archive/refs/tags/ignition-math6_6.8.0.tar.gz"
+    sha256: "07c4fa9eeaa51fe2b4ee2855ea69bb16571d7e6c315b8b0726fc4383b64303a1"
+  #"6.9.0":
```

Comment:
Why the latest version is disabled?

Reply:
You could remove it for now, merge this PR, and open a new one in the future only bumping the version.

Reply:
I'd rather wait than repeat the extra work. Besides there are 3 other PR ([here](https://github.com/conan-io/conan-center-index/pulls/ggulgulia))that depends on #9491 and I've already explained in detail in [this review comment](https://github.com/conan-io/conan-center-index/pull/9491#discussion_r817920906) and in all the issues and PRS, why I need an updated ignition-cmake. 

If possible, you can already review and approved the PR related to here(#9491)ignition-cmake :wink:

Reply:
Great, thank you for your clarification. Usually we only push the latest version to avoid more and more storage/ci time. Only when someone really need an older version, we add it.